src/plot_api/plot_api.js

```diff
@@ -1632,26 +1639,6 @@ function _restyle(gd, aobj, traces) {
         }
     }
 
-    // check axes we've flagged for possible deletion
-    // flagAxForDelete is a hash so we can make sure we only get each axis once
-    var axListForDelete = Object.keys(flagAxForDelete);
```

Comment:
This isn't really related to `Plotly.react` but noticed as I was working here: post #2227 we don't need to delete unused axes anymore - in fact we don't *want* to.

Reply:
Oh but it's not self-contained here - I need to get rid of `flagAxForDelete` above...

Reply:
finished deleting in https://github.com/plotly/plotly.js/pull/2341/commits/aab4f11ff1200b8a58d287a03670034354a6b060

Reply:
This has most probably caused:
https://github.com/plotly/plotly.js/issues/2392